debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -531,7 +531,7 @@ public Optional<SlotCreationResult> createReplicationSlot() throws SQLException
                     slotName,
                     tempPart,
                     plugin.getPostgresPluginName(),
-                    lsnType.getLsnTypeName(),
+                    lsnType.getLsnTypeName().equalsIgnoreCase("SEQUENCE") ? "" : "HYBRID_TIME",
```

Comment:
Lets add a comment why we are sending empty. For backward compatibility. We can file a ticket, when default changes this also has to change.